README.md

```diff
@@ -21,33 +21,33 @@ to generate systemd service configuration to run a container via
 
 ### Testing
 
-RPM packages for the Hirte project are available on
-[hirte-snapshot](https://copr.fedorainfracloud.org/coprs/mperina/hirte-snapshot/)
-COPR repo. To install hirte packages on your system please add that repo using:
+RPM packages for the BlueChi project are available on
+[bluechi-snapshot](https://copr.fedorainfracloud.org/coprs/mperina/hirte-snapshot/)
```

Comment:
I'd leave it as `hirte-snapshot` until we resolve if to rename or switch to packit